misc/python/materialize/feature_benchmark/benchmark_versioning.py

```diff
@@ -10,7 +10,7 @@
 
 from materialize import MZ_ROOT
 
-FEATURE_BENCHMARK_FRAMEWORK_VERSION = "1.3.0"
+FEATURE_BENCHMARK_FRAMEWORK_VERSION = "1.4.0"
```

Comment:
Not sure if this change warrants upgrading the version.

Reply:
In metabase can we easily merge multiple versions? If yes, then the bump is fine and we can just display them together for now.

Reply:
Not really. Reverting the version upgrade